src/k2/dto/SKVRecord.cpp

```diff
@@ -32,7 +32,7 @@ namespace dto {
 
 void SKVRecord::skipNext() {
     if (fieldCursor >= schema->fields.size()) {
-        throw SKVRecordException("Schema not followed in record serialization");
+        throw NoFieldFoundException();
```

Comment:
Why not have the string to give a little more info?

Reply:
added